src/openbench/pipeline/keyword_boosting/openai.py

```diff
@@ -0,0 +1,71 @@
+from pathlib import Path
+from typing import Callable, Optional
+
+from pydantic import Field
+
+from ...engine import OpenAIApi, OpenAIApiResponse
+from ...pipeline import Pipeline, PipelineConfig, register_pipeline
+from ...pipeline_prediction import Transcript
+from ...types import PipelineType
+from .common import BoostingOutput
+
+
+TEMP_AUDIO_DIR = Path("temp_audio_dir")
+
+
+class OpenAIBoostingPipelineConfig(PipelineConfig):
+    model_version: str = Field(
+        default="whisper-1",
```

Comment:
Are you thinking about replacing the API based config with the open-source project or keeping them separate?